lib/array_enum.rb

```diff
@@ -9,21 +9,26 @@ module ArrayEnum
   def array_enum(definitions)
     definitions.each do |attr_name, mapping|
       attr_symbol = attr_name.to_sym
+      mapping_hash = ActiveSupport::HashWithIndifferentAccess.new(mapping)
+
+      define_singleton_method(attr_name.to_s.pluralize) do
```

Comment:
could you add `require "active_support/core_ext/string/inflections"` on top of file?